src/main/java/Elon/Deadline.java

```diff
@@ -0,0 +1,25 @@
+package Elon;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+public class Deadline extends Task {
+    protected boolean isDone;
+    protected LocalDate by;
+    public Deadline(String description, boolean isDone, LocalDate by) {
+        super(description, isDone);
+        this.by =by;
```

Comment:
Please use whitespace between operands, i.e. `this.by = by` instead of `this.by =by`.